.github/ISSUE_TEMPLATE/--feature-request.yaml

```diff
@@ -1,7 +1,7 @@
 name: Feature request
 description: Suggest a feature to improve Keploy
 title: "[feature]: "
-labels: [feature]
+labels: [Enhancement]

```

Comment:
Let's not change this as well!

Reply:
Same , we don't have ``feature`` label in any of repository.